### DISCUSSION TITLE
Including additional files in gen?

### DISCUSSION THREAD

**Discussion by knro** [dashingsoft/pyarmor#1657] [Q&A] [answered]
I'm using latest 8.4.6 (pro) and I have a package structure that includes additional files:

root:
  my_package:
      init.py
      foo.py
      database.sqlite
  pyproject.toml
  MAINFEST.in

Now I included already in the MANIFEST.in this:

include my_package/database.sqlite

so I expected when I run this:

`pyarmor gen -O packaging -r -i my_package`

that database.sqlite would be also copied to packaging/my_package along with the obfuscated files, but this is not happening. Any ideas what I am doing wrong?

**Comment by jondy** [comment, marked as answer]
Please check
https://pyarmor.readthedocs.io/en/latest/tutorial/obfuscation.html#copying-package-data-files
Or write your own `post_build` plugin to copy data files

**Comment by knro** [comment]
Thank you! 

`pyarmor cfg data_files=*`

Did the trick
